backend/src/main/java/codezap/member/controller/SpringDocMemberController.java

```diff
@@ -343,4 +352,44 @@ public interface SpringDocMemberController {
             )
     )
     void logout(HttpServletResponse response);
+
+
+    @SecurityRequirement(name = "쿠키 인증 토큰")
+    @Operation(summary = "회원 정보 조회", description = "회원의 정보(이메일, 닉네임)을 조회합니다.")
+    @ApiResponse(responseCode = "200", description = "회원 정보 조회 성공")
+    @ApiErrorResponse(status = HttpStatus.UNAUTHORIZED, instance = "/members/1", errorCases = {
+            @ErrorCase(description = "쿠키 값 오류", exampleMessage = "인증에 실패했습니다.")
+    })
+    @ApiErrorResponse(status = HttpStatus.FORBIDDEN, instance = "/members/1", errorCases = {
+            @ErrorCase(description = "조회하려는 회원에 대해 권한이 없는 경우", exampleMessage = "권한이 없어 식별자 1에 해당하는 회원 정보를 조회할 수 없습니다.")
+    })
+    @ApiErrorResponse(status = HttpStatus.NOT_FOUND, instance = "/members/1", errorCases = {
+            @ErrorCase(description = "조회하려는 id 값인 회원이 없는 경우", exampleMessage = "식별자 1에 해당하는 회원이 존재하지 않습니다.")
+    })
+    ResponseEntity<FindMemberResponse> findMember(MemberDto memberDto, Long id);
+
+    @SecurityRequirement(name = "쿠키 인증 토큰")
+    @Operation(summary = "회원 정보 수정", description = "회원의 정보(사용자명, 비밀번호)을 수정합니다.")
+    @ApiResponse(responseCode = "200", description = "회원 정보 수정 성공")
```

Comment:
⚡ 성공 케이스일 때 어떤 response가 반환되는지 명시해주세요
```suggestion
    @ApiResponse(responseCode = "200", description = "회원 정보 수정 성공",
          content = {@Content(schema = @Schema(implementation = UpdateMemberProfileResponse.class))})
```